test/test_transformers.py

```diff
@@ -1144,7 +1181,7 @@ def test_fused_sdp_choice(self, type: str):
             value = value.view(batch_size, -1, num_heads, head_dim).transpose(1, 2)
             key = key.view(batch_size, -1, num_heads, head_dim).transpose(1, 2)
 
-            if SM80OrLater:
+            if SM80OrLater and not type == "nested":
```

Comment:
flash-attention for nested tensors is currently turned of because of cuda memory issues